benchopt/plotting/plot_objective_curve.py

```diff
@@ -66,16 +64,10 @@ def plot_objective_curve(df, obj_col='objective_value', plotly=False,
         max_f_0 = df[df['stop_val'] == 1][obj_col].max()
         df.loc[:, obj_col] /= max_f_0
 
-    fig = get_figure(plotly)
+    fig = plt.figure()
```

Comment:
also remove all import from helper compat, that are only here to make it work with plotly.